services-js/commissions-app/src/client/common/TextInput.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Head from 'next/head';
+import ReachDOM from 'react-dom';
+
+export class TextInput extends React.Component {
+  render() {
+    return (
+      <input
```

Comment:
You’re missing the `<label>` element and other things.

Remember we talked about how this component should have the whole `<div className="txt">…</div>` piece.